### PR TITLE
Add missing RT deps for patch-u-boot-env

### PR DESCRIPTION
**Pull request by fmoessbauer**
cc @gylstorffq

### CODE REVIEW

**Comment by fmoessbauer** [comment]
@jan-kiszka please review

File: recipes-bsp/u-boot/u-boot-iot2050.inc [hunk 1]
```diff
@@ -46,6 +46,10 @@ TEMPLATE_VARS += "FIRMWARE_SECURE_VER"
 
 U_BOOT_CONFIG_PACKAGE = "1"
 
+# break dependency chain by adding support for runtime provides / depends
+# TODO: once this is globally enabled in ISAR, remove this line
+RPROVIDES += "${PROVIDES}"
+
```
Comment (jan-kiszka) [inline recipes-bsp/u-boot/u-boot-iot2050.inc:52]: IIUC, we should wait for Isar to pick up the generic solution.

Comment (fmoessbauer) [reply recipes-bsp/u-boot/u-boot-iot2050.inc:52]: We now have the support in ISAR. Once we have the update here in meta-iot2050 as well, I will update this.
A similar change also needs to be done in `meta-coral` to reduce the number of components that are re-build on changes.

**Comment by jan-kiszka** [comment]
You may already want to look into rebasing over #420.

**Comment by fmoessbauer** [comment]
> You may already want to look into rebasing over #420.

Just tested that locally and can confirm it works. For Github I would like to wait till #420 is integrated to avoid including a lot of changes in the diff which might distract from the actual fix when reviewing.

**Review by jan-kiszka — APPROVED** [review]

**Pull request merged**